src/use_case/new_game/NewGameInputBoundary.java

```diff
@@ -1,4 +1,5 @@
 package use_case.new_game;
 
 public interface NewGameInputBoundary {
+    void execute(NewGameInputData newGameInputData);
```

Comment:
we should probably use more specific names instead of just execute though it's fine for now just for the future